### PR TITLE
Update index to remove how-to, and add additional reference

### PR DESCRIPTION
**Pull request by danrosenthal**

### CODE REVIEW

**Comment by danrosenthal** [comment]
@cromwellryan this is ready for review.

File: index.md [hunk 1]
```diff
@@ -32,6 +31,7 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
 
 **Additional Resources:**
 
+* [Practical Object Oriented Programming in Ruby][] (“sb-bookshelf” in Dropbox)
```
Comment (danrosenthal) [inline index.md]: Such a great way to do markdown urls!

File: index.md [hunk 2]
```diff
@@ -11,13 +11,12 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
...
 **Training:**
 
-* [CodeSchool: Try Git][]
-* [CodeSchool: Git Real 2][]
-* [CodeSchool: Git Real][]
-* [Sparkbox how_to][]: your first experience with Git
-* [More Git][]: more Git practice
-* [CodeSchool: Try Ruby][]
-* [Ruby setup][]: set up your machine for running Ruby
+* ~~[CodeSchool: Try Git][]~~
```
Comment (cromwellryan) [inline index.md]: Can we remove the strikethroughs here @danrosenthal?

File: projects/proj-more-git.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 # Project: More Git
 
-1. Drew's forked repo of the new Sparkbox site's content prototype to your personal account: <https://github.com/drewclemens/sparkbox-website>
-2. Add your own profile page to the site
-3. Submit a pull request for review
\ No newline at end of file
+1. Make a dotfiles directory and store any relevant dotfiles that you'll make changes to inside of that directory (e.g. .bash_profile, .bashrc, .gitconfig, .gitignore).
+2. Initialize a new repository inside of your dotfiles directory.
+3. Create symlinks for any dotfiles that you've moved from your root.
```
Comment (cromwellryan) [inline projects/proj-more-git.md]: _root_ should probably be home directory or `$HOME`

File: projects/proj-more-git.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
...
+4. Add customizations to your dotfiles as you see fit.
+5. Push dotfiles directory to GitHub.
```
Comment (cromwellryan) [inline projects/proj-more-git.md]: _directory_ should probably be _repo_ or _repository_.  Something like: "Push your `dotfiles` repository to Github."

File: index.md (hunk 2 above)
Comment (cromwellryan) [inline index.md]: Ah, I see what happened.  Hey @laurendorman, love that you're using the curriculum to track your progress, but we probably shouldn't push it to this remote as happened in a0bb350.  I'll show you how to remove that commit tomorrow.

**Comment by danrosenthal** [comment]
@cromwellryan I have made your requested changes. @laurendorman and @kaseybon should do a new git pull for the latest version. Also, if you want to strikethrough any items that you have completed, be sure to do it on your own branch and don't push to master.

**Comment by laurendorman** [comment]
My mistake @cromwellryan, I forked the repo but did not change the origin before pushing. My local repo is now linked to my forked repo. Thanks for the fix @danrosenthal!

**Pull request merged**